<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutUsWindow</class>
 <widget class="QMainWindow" name="AboutUsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string>QMainWindow {
    background-color: #F5F5F5;
    font-family: 'Segoe UI', sans-serif;
}

QPushButton {
    background-color: #0078D7;
    border: none;
    border-radius: 4px;
    color: white;
    padding: 10px;
    min-width: 150px;
}

QPushButton:hover {
    background-color: #005FAC;
}

QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333;
}

QScrollArea {
    border: none;
}

QTextEdit {
    border: 1px solid #ccc;
    font-size: 14px;
    color: #333;
    padding: 10px;
    background-color: #F5F5F5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="aboutUsTitleLabel">
      <property name="text">
       <string>关于我们</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>750</width>
         <height>500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>500</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="scrollAreaLayout">
        <item>
         <widget class="QLabel" name="teamMemberLabel1">
          <property name="text">
           <string>张三 - 项目经理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel2">
          <property name="text">
           <string>李四 - 后端开发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel3">
          <property name="text">
           <string>王五 - 前端开发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel4">
          <property name="text">
           <string>赵六 - 测试工程师</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel5">
          <property name="text">
           <string>孙七 - UI设计师</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel6">
          <property name="text">
           <string>周八 - 数据分析师</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel7">
          <property name="text">
           <string>吴九 - 系统架构师</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="teamMemberLabel8">
          <property name="text">
           <string>郑十 - 流程优化专家</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>返回主菜单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
